src/script/components/UserSearchableList.tsx

```diff
@@ -165,7 +177,7 @@ const UserSearchableList: React.FC<UserListProps> = props => {
       users={foundUserEntities()}
       selectedUsers={selectedUsers}
       highlightedUsers={highlightedUsers}
-      onSelectUser={observables.selected ? toggleUserSelection : undefined}
+      onSelectUser={!!selectedUsers ? toggleUserSelection : undefined}
```

Comment:
```suggestion
      {...(!!selectedUsers && {
        onSelectUser: toggleUserSelection
      })
```

Make smth like that :)